test/topology/test_change_ip.py

```diff
@@ -28,3 +29,30 @@ async def test_change_two(manager, random_tables):
     await manager.server_start(s_2)
     await table.add_column()
     await random_tables.verify_schema()
+
+
+@pytest.mark.asyncio
+async def test_change_all(manager, random_tables):
+    """Add a node, stop an original node, add a column"""
+    servers = await manager.running_servers()
+    table = await random_tables.add_table(ncolumns=5)
+    stops = []
+    for server in servers:
+        logger.info("Gracefully stopping server %s to change ip", server.server_id)
+        stops.append(manager.server_stop_gracefully(server.server_id))
+    await asyncio.gather(*stops)
+
+    starts = []
+    for server in servers:
+        logger.info("Starting server %s", server.server_id)
+        await manager.server_change_ip(server.server_id)
+        starts.append(manager.server_start(server.server_id))
+    await asyncio.gather(*starts)
+    # Despite hacks in server_start() to refresh the connection list, if
+    # we shut down the entire cluster, they don't work. TODO: investigate
+    # why. Meanwhile, without a manual close/connect the test tear down
+    # fails to drop the keyspace and the post-test sanity check fails
+    manager.driver_close()
+    await manager.driver_connect()
```

Comment:
If you want to avoid a driver update for each IP change, perhaps you could add a flag to `manager.server_change_ip(update_driver=True)` and then call `manager.update_driver()` manually.